README.md

```diff
@@ -1,155 +1,3 @@
-
-1. `usb-test` : Preflight check that probes for USB connected devices, that returns:
-    - `USB_CONNECTED` if a USB device is attached to the dedicated slot
-    - `USB_NOT_CONNECTED` if no USB is attached
-    - `USB_CHECK_ERROR` if an error occurred during pre-flight
-
-2. `disk-test`: Preflight check that checks for LUKS-encrypted volume that returns:
-    - `USB_ENCRYPTED` if a LUKS volume is attached to sd-devices
-    - `USB_ENCRYPTION_NOT_SUPPORTED` if a LUKS volume is not attached or there was any other error
-    - `USB_DISK_ERROR`
-
-3. `printer-test`: prints a test page that returns:
-    - `ERROR_PRINTER_NOT_FOUND` if no printer is connected
-    - `ERROR_PRINTER_NOT_SUPPORTED` if the printer is not currently supported by the export script
-    - `ERROR_PRINTER_DRIVER_UNAVAILABLE` if the printer driver is not available
-    - `ERROR_PRINTER_INSTALL` If there is an error installing the printer
-    - `ERROR_PRINT` if there is an error printing
-
-4. `printer`: sends files to printer that returns:
-    - `ERROR_PRINTER_NOT_FOUND` if no printer is connected
-    - `ERROR_PRINTER_NOT_SUPPORTED` if the printer is not currently supported by the export script
-    - `ERROR_PRINTER_DRIVER_UNAVAILABLE` if the printer driver is not available
-    - `ERROR_PRINTER_INSTALL` If there is an error installing the printer
-    - `ERROR_PRINT` if there is an error printing
-
-5. `disk`: sends files to disk that returns:
-    - `USB_BAD_PASSPHRASE` if the luks decryption failed (likely due to bad passphrase)
-    - `ERROR_USB_MOUNT` if there was an error mounting the volume (after unlocking the luks volume)
-    - `ERROR_USB_WRITE` if there was an error writing to disk (e.g., no space left on device)
-
-### Export Folder Structure
-
-When exporting to a USB drive, the files will be placed on the drive as follows: The root of the USB drive will contain one `sd-export-[timestamp]` folder, where `[timestamp]` is in the format `YYYYMMDD-hhmmss`. This folder will contain a subfolder `export_data`, which will contain the exported file with its original name as submitted by the source. For example:
-
-```
-.
-
-└── sd-export-20200116-003153
-    └── export_data
-        └── secret_memo.pdf
-```
-
-To support multiple files, in the long term, we are planning to use a folder structure similar to the following, where the journalist designation for a source is used for folder names and message/reply file names.
-
-
-```
-.
-
-├── cytotoxic-payer
-│   ├── 1-cytotoxic-payer-msg
-│   │   └── 1-cytotoxic-payer-msg.txt
-│   ├── 2-cytotoxic-payer-msg
-│   │   └── 2-cytotoxic-payer-msg.txt
-│   └── 3-cytotoxic-payer-doc
-│   │   └── interesting_file.doc
-├── grandiloquent-pasteboard
-│   └── 1-grandiloquent-pasteboard-doc
-│   │   └── questionable_file.pdf
-└── snug-seek
-```
+This repository has been merged into [`securedrop-client`](https://github.com/freedomofpress/securedrop-client).
```

Comment:
Is there any reason not to link directly here to the `export/` directory?

Reply:
Because this repository will get archived after this is merged, I think linking directly to the client repo is going to be the most stable URL in the long-run, i.e. the export/ directory could be refactored/renamed, but the repository itself should last or redirect to the new target.

Reply:
That's persuasive.  :-)